src/app/pages/PerpetualPage/components/SlippageForm/index.tsx

```diff
@@ -11,6 +11,18 @@ type SlippageFormProps = {
   onChange: (slippage: number) => void;
 };
 
+const SLIPPAGE_MIN = 0.001;
+const SLIPPAGE_MAX = process.env.REACT_APP_NETWORK === 'mainnet' ? 0.01 : 0.02;
```

Comment:
I see we use this condition on 2 other places in the dapp, extract it to a global constant.